src/freertos_drivers/common/WiFiInterface.hxx

```diff
@@ -0,0 +1,377 @@
+/** \copyright
```

Comment:
This is an overall comment.  I think the add_ssid_profile() and setup_access_point() forms are less elegant than ssid_profile_add() and access_point_setup.  In other words, lead with the subject, follow with the action.  We did not consistently follow this with the CC32xx, but looking back, we probably should have.